AirSim/docs/python.md

```diff
@@ -0,0 +1,3 @@
+# Using Python APIs for AirSim
+
+This document is now merged with [general APIs document](apis.md).
```

Comment:
do we want to add this to our repo documentation base?

Reply:
Probably not

Reply:
Addressed 0e7b80f